src/pages/workspace/WorkspaceInviteMessagePage.js

```diff
@@ -71,188 +71,141 @@ const defaultProps = {
     invitedEmailsToAccountIDsDraft: {},
 };
 
-class WorkspaceInviteMessagePage extends React.Component {
-    constructor(props) {
-        super(props);
+function WorkspaceInviteMessagePage(props) {
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
 
-        this.sendInvitation = this.sendInvitation.bind(this);
-        this.validate = this.validate.bind(this);
-        this.openPrivacyURL = this.openPrivacyURL.bind(this);
-        this.debouncedSaveDraf = _.debounce((newDraft) => {
-            Policy.setWorkspaceInviteMessageDraft(this.props.route.params.policyID, newDraft);
-        }, 2000);
-        this.state = {
-            welcomeNote: this.props.workspaceInviteMessageDraft || this.getDefaultWelcomeNote(),
-        };
-    }
-
-    componentDidMount() {
-        if (_.isEmpty(this.props.invitedEmailsToAccountIDsDraft)) {
-            Navigation.goBack(ROUTES.WORKSPACE_INVITE.getRoute(this.props.route.params.policyID), true);
-            return;
-        }
-        this.focusWelcomeMessageInput();
-    }
-
-    componentDidUpdate(prevProps) {
-        if (!prevProps.isFocused && this.props.isFocused) {
-            this.focusWelcomeMessageInput();
-        }
+    const {inputCallbackRef} = useAutoFocusInput();
 
-        if (
-            !(
-                (prevProps.preferredLocale !== this.props.preferredLocale || prevProps.policy.name !== this.props.policy.name) &&
-                this.state.welcomeNote === Localize.translate(prevProps.preferredLocale, 'workspace.inviteMessage.welcomeNote', {workspaceName: prevProps.policy.name})
-            )
-        ) {
-            return;
-        }
-        this.setState({welcomeNote: this.getDefaultWelcomeNote()});
+    if (_.isEmpty(props.invitedEmailsToAccountIDsDraft)) {
```

Comment:
Whenever we complete inviting a member, it redirects to invite page after this change. Earlier it goes to Members page.

We should do this only on mount with useEffect to preserve the existing behaviour.